assets/js/i18n/LocaleSwitcher.js

```diff
@@ -57,7 +57,7 @@ export default function LocaleSwitcher ({ closeModal }) {
         </form>
       </ModalBody>
       <ModalFooter>
-        <button className='action-button button-gray' onClick={closeModal}>
+        <button className='action-button button-gray close-button' onClick={closeModal}>
```

Comment:
The user doesn't need to click "Close" to switch locales. Clicking the language refreshes the page.

Please nix this class. As before: class names should be for CSS.